### PR TITLE
feat(mcdu): scratchpad clear on clr held

### LINKED ISSUE #5080: MCDU CLR button hold behaviour incorrect
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-06-09T07:52:45+00:00",
  "ref": "master",
  "sha": "e3cda27aa50d6a8173363ccb7ae256888dc67842",
  "actor": "davidwalschots",
  "event_name": "manual"
}
```

### Describe the bug

Holding the CLR button on the MCDU only clears the last digit, same as a brief press

### Expected behavior

Holding the CLR button on the MCU for 3 seconds should clear the whole scratchpad
Addition: On NEOs and never CEOs it's ~1 second.

### Steps to reproduce

1. Type random stuff on the MCDU
2. Hold CLR and observe only the last digit is erased

### References (optional)

image

### Additional info (optional)

_No response_

### Discord Username (optional)

raelias #5157

### PR DESCRIPTION
**Pull request by MisterChocker**
Fixes #5080

## Summary of Changes

- Added clear scratchpad when clear button held for ~1sec.

## Additional context

[Source](https://github.com/flybywiresim/a32nx/pull/4996)

Discord username (if different from GitHub): DerL30N#3751

## Testing instructions

Press and hold the clr button on the mcdu and see the entire scratchpad being cleared.

Situations: (Press and hold)
_`INITIAL SCRATCHPAD STATE` -> `SCRATCHPAD WHEN CLR PRESSED` -> `SCRATCHPAD WHEN HELD`_
- `FBW RULE` -> `FBW RUL` -> ` `
- `A` -> ` ` -> ` `
- `CLR` -> ` ` -> ` `

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by raeliass** [comment]
CLR should be held for 3 seconds or more, check this report references https://github.com/flybywiresim/a32nx/issues/5080

**Comment by tracernz** [comment]
> CLR should be held for 3 seconds or more, check this report references #5080

We got videos from our engineers and pilots for this one. It's definitely 1 second on both Honeywell and Thales, despite what the documents say.

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js [hunk 1]
```diff
@@ -101,10 +101,12 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
             this.tryShowMessage();
         };
         this.onClrHeld = () => {
-            if (this.inOut !== "" && this.inOut !== FMCMainDisplay.clrValue) {
+            if (this.inOut === FMCMainDisplay.clrValue) {
+                this.inOut = "";
```
[the remark this answers is not in the document]
Comment (MisterChocker) [reply flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:105]: We actually have a reference of that and this is what the reference shows

**Review by tracernz — APPROVED** [review]
LGTM

**Comment by marcman86** [comment]
Quality Assurance Tester Report
Discord:marcman86#4907
Object of testing: #5046
Tier of Testing: 1
Date: 10/06/2021

Testing Process: 
- Spawned C&D at EDDP
- Turned on bat. and ext. power
- Press CLR to clear GPS PRIMARY LOST
- Type in some digits
- Press CLR to clear single digits
- Press and hold CLR to clear all digits
- Align ADIRS
- Press and hold CLR to clear GPS PRIMARY

Negatives: I dont know the behaviour in real life but when pressed and hold first the last digit clears and then after a second the rest. This looks awkward. When hold down should the single clear be inhibit?
Testing Results: Passed, when found a reference for the issue.
Conclusions: Works in general
Media: -
Hope this helps :-)

**Comment by MisterChocker** [comment]
@marcman86 IRL when pressed and held, first the last number will disappear and then the rest.

**Comment by raeliass** [comment]
> > CLR should be held for 3 seconds to clear the whole scratchpad
> > image
> 
> You already commented on this post and we already responded to you - the documents are incorrect and we have gotten videos from both mcdu models and for both it is one second.

Oh I didn't see it, ok then

**Comment by FlightSimmer7700** [comment]
Quality Assurance Tester

Discord: FlightSimmer7700#3468
Object of testing: #5046
Tier of Testing: 1
Date: 11/6/21
Testing Process:
Full flight while using it the whole time with a mix of numbers and letters
Negatives: N/A
Testing Results: Passed

Conclusions: LGTM

**Pull request merged**